prover/lang/kore-lang.md

```diff
@@ -552,12 +539,10 @@ Substitution: Substitute term or variable
   rule substPatternsMap(.Patterns, SUBST) => .Patterns
 
   // the Bool param: avoid capture
-  syntax Patterns ::= Patterns "[" Bool "," Pattern "/" Pattern "]" [function]
   syntax Patterns ::= Patterns "[" Pattern "/" Pattern "]" [function]
 
-  rule Ps[X/V] => Ps[false, X/V]
-  rule (BP, BPs)[CA, X/V] => #subst(CA,BP,X,V), (BPs[CA, X/V])
-  rule .Patterns[_, X/V] => .Patterns
+  rule (BP, BPs)[X/V] => subst(BP,X,V), (BPs[X/V])
```

Comment:
This notation `P[X/V]` is different from what we usually use in the papers: `P[V/X]`, but it's OK. I suggest a comment here.

Reply:
I'd honestly like to get rid of the mix fix notation  for substitution completely. I find it really confusing.